cmd2/utils.py

```diff
@@ -144,3 +144,25 @@ def is_text_file(file_path: str) -> bool:
             pass
 
     return valid_text_file
+
+
+def remove_duplicates(list_to_prune: List) -> List:
+    """
+    Removes duplicates from a list while preserving order of the items
+    :param list_to_prune: the list being pruned of duplicates
+    :return: The pruned list
+    """
+    temp_dict = collections.OrderedDict()
+    for item in list_to_prune:
+        temp_dict[item] = None
+
+    return list(temp_dict.keys())
+
+
+def alphabetical_sort(list_to_sort: List[str]) -> List[str]:
+    """
+    Sorts a list of strings alphabetically
+    :param list_to_sort: the list being sorted
+    :return: the sorted list
+    """
+    return sorted(list_to_sort, key=str.casefold)
```

Comment:
Consider doing full-blown Unicode normalization and casefolding here.  It is significantly more complex, but this StackOverflow article has a good explanation of how exactly it should be done: https://stackoverflow.com/questions/40348174/should-i-use-python-casefold

The associated utility functions would likely be useful elsewhere as well.